src/python/grapl_analyzerlib/grapl_analyzerlib/scratch_test.py

```diff
@@ -0,0 +1,307 @@
+import json
```

Comment:
(what exactly is this testing)

Reply:
Nothing, I plan to delete this file. It's just iteration smoke testing to make sure I don't break things as I go.